static/js/apps/explore/app.tsx

```diff
@@ -128,339 +94,228 @@ export function App(): JSX.Element {
     };
   }, []);
 
-  useEffect(() => {
-    setLoadingStatus("loading");
-    (async () => {
-      let place = getSingleParam(hashParams["p"]);
-      let cmpPlace = getSingleParam(hashParams["pcmp"]);
-      let topic = getSingleParam(hashParams["t"]);
-      let cmpTopic = getSingleParam(hashParams["tcmp"]);
-      let placeType = getSingleParam(hashParams["pt"]);
-      let paramQuery = getSingleParam(hashParams["q"]);
-      let classificationsStr = getSingleParam(hashParams["cls"]);
-      const origQuery = getSingleParam(hashParams["oq"]);
-      const dc = getSingleParam(hashParams["dc"]);
-      const svg = getSingleParam(hashParams["svg"]);
-      const disableExploreMore = getSingleParam(hashParams["em"]);
-      const nlFulfillment = getSingleParam(hashParams["nl"]);
-
-      // Do detection only if `q` is set (from search box) or
-      // if `oq` is set without accompanying place and topic.
-      if (paramQuery || (origQuery && !place && !topic)) {
-        if (!paramQuery) {
-          // This should only be set once at the very beginning!
-          paramQuery = origQuery;
-        }
-        setQuery(paramQuery);
-        const detectResp = await fetchDetectData(
-          paramQuery,
-          savedContext.current,
-          dc
-        );
-        if (!detectResp) {
-          setLoadingStatus("fail");
-          return;
-        }
-        savedContext.current = detectResp["context"] || [];
-        if (_.isEmpty(detectResp["entities"])) {
-          setLoadingStatus("fail");
-          return;
+  const exploreContext = _.isEmpty(savedContext.current)
+    ? null
+    : savedContext.current[0]["insightCtx"];
+  if (loadingStatus === LoadingStatus.FAILED) {
+    return (
+      <Container className="explore-container">
+        <div>
+          <div id="user-message">Sorry, could not complete your request.</div>
+          {query && (
+            <SearchSection
+              query={query}
+              debugData={debugData}
+              exploreContext={exploreContext}
+            />
+          )}
+        </div>
+      </Container>
+    );
+  }
+  if (loadingStatus === LoadingStatus.LOADING) {
+    return (
+      <Container className="explore-container">
+        <div>
+          <Spinner isOpen={true} />
+        </div>
+      </Container>
+    );
+  }
+  if (loadingStatus === LoadingStatus.SUCCESS && chartData) {
+    const childPlaceType = !_.isEmpty(chartData.childPlaces)
+      ? Object.keys(chartData.childPlaces)[0]
+      : "";
+    const placeUrlVal = (
+      exploreContext?.entities || [chartData.place.dcid]
+    ).join(DELIM);
+    const topicUrlVal = (exploreContext?.variables || []).join(DELIM);
+    const relatedPlaceTopic = _.isEmpty(chartData.mainTopic)
+      ? {
+          dcid: exploreContext?.variables,
```

Comment:
This can be a list?  do you mean to set it to just dcid?

Reply:
this was following what the app used to do: https://github.com/datacommonsorg/website/blob/65909ffa310fbb6ec191693f7dd2f6832c969e5f/static/js/apps/explore/app.tsx#L430-L438

is taht something we wanted to change?

Reply:
oh, actually, need to use topicUrlVal, good catch